OpenSim/Common/Exception.h

```diff
@@ -67,14 +110,31 @@ class OSIMCOMMON_API Exception  : public std::exception {
     std::string _file;
     /** Line number at which the error occurred. */
     int _line;
+    /** Delimiter/separator used to separate different parts of the message.  */
+    static const std::string _msgSeparator;
```

Comment:
Shrinidhi, please make the data members private if they should be. This is the time to do it when a reviewer has spotted the problem for you. It will take much less time to do it than to discuss it further.